ottoVonWinstonIV.py

```diff
@@ -67,11 +67,23 @@ async def recceInStockMessage():
             else:
                 print(itemCatalog["RECCE Rig"][0] + "\n\n")
 
+    async def fridaySpecial():
+        channel = client.get_channel(int(os.getenv('GAME_CHAT_CHANNEL')))
+        fridaySpecialPath = os.getcwd() + "\FridaySpecial.mov"
+        fridaySpecialFile = discord.File(fridaySpecialPath, filename = "FridaySpecial.mov")
+        while(1):
+            await asyncio.sleep(60*5)
+            currentTime = time.asctime()
+            if (currentTime.tm_wday == 4) and (currentTime.tm_hour == 9) and (currentTime.tm_min < 10):

```

Comment:
Wouldn't Friday be tm_wday == 5?

[sun, mon, tue, wed, thur, fri, sat]

Reply:
https://docs.python.org/3/library/time.html#time.gmtime

![image](https://user-images.githubusercontent.com/129230471/232249543-f7f6bea4-1301-4b19-91ea-2f17985a19f1.png)

![image](https://user-images.githubusercontent.com/129230471/232249554-72a04f2a-78ae-436d-87d9-5856e433cdb6.png)

Reply:
I have been mislead by geeks for geeks, and I am sorry.